app/agents/validator.py

```
@@ -0,0 +1,272 @@
+"""
+Agente Validator - Valida datos específicos del usuario
+Integra con las funciones de validación existentes en utils/validators.py
+"""
+
+import re
+from typing import Dict, Any, List, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)